### DISCUSSION TITLE
Mandeeps Live Content - Cannot manage photos

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1937] [General]
We can upload new images.  But we cannot edit, move or delete images.
The editing icons show when the admin bar on the left is open, but they disappear when the menu is closed.

**Comment by ashishpachauriap** [comment]
We have already logged this issue to be resolved in the next update of Live Content.

**Reply by scottw12** [reply]
Is there any workaround until you do get this fixed?

**Reply by ashishpachauriap** [reply]
**Workaround:**
You can edit the element by link like http://dnndev.120.com/live-content/ctl/manage/mid/360#/edit/elementId/1/type/ePicture
where 360 is your **ModuleID** and 1 is **ElementID**.
You can find **ModuleID** and **ElementID** by inspecting the image.

**Reply by scottw12** [reply]
IS there any update on this or did it just go to the dead pile? I do have the latest version of live content installed.
Live Content v4.7.8
I have not renewed my mandeeps licenses due to some of these issues.

**Reply by scottw12** [reply]
By the way, your workaround does not work.

**Reply by Gaganbird** [reply]
It will be available in the next major update.

**Comment by scottw12** [comment]
Another problem we are having that is related is Live-Visualizer and editing/deleting icons.  Security is non-functional and the edit and delete icons are presented to unregistered users. 

This is not happening on every page where we use visualizer as seen in the 2 pages listed below. 

https://dev.jcesba.org/about-us/who-we-are/board-of-directors
https://dev.jcesba.org/events/calendar
 
We were going to do a public launch on Thursday of this week. A work around for these issues would be greatly appreciated.

**Reply by ashishpachauriap** [reply]
Please enable cache on user level in your visualizer and try again.
Navigate to Manage > edit visualizer > Settings > Cache > Enable User cache
